### PR TITLE
note listing page UI

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by josephmathew900**
Fixes #1

### CODE REVIEW

**Comment by josephmathew900** [comment]
note-list-ui

**Comment by josephmathew900** [comment]
note-list-ui

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from "react";
-import { Checkbox } from "neetoui";
+import { Checkbox, Tooltip, Button, Badge, Avatar } from "neetoui";
+import dayjs from "dayjs";
+import tagOptions from "constants/tagOptions";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:4]: Please rename the imported constant as `TAG_OPTIONS`.
Please refer [naming constants](https://github.com/airbnb/javascript#naming--uppercase) for more details.

Comment (karthiknmenon) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:4]: Also, please don't create aliases for constants. Rather, please create a `constants` file per feature. Like the `constants` file for notes feature can be inside the directory `app/javascript/src/components/Dashboard/Notes`.


However, you can create an alias for `constants` when you consider global constants that will be used throughout the application or at multiple locations.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
-import notesApi from "apis/notes";
+// import notesApi from "apis/notes";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:2]: Please remove all comments.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 3]
```diff
@@ -24,8 +27,8 @@ const Notes = () => {
   const fetchNotes = async () => {
     try {
       setLoading(true);
-      const response = await notesApi.fetch();
-      setNotes(response.data);
+      // const response = await notesApi.fetch();
+      // setNotes(response.data);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:31]: Please remove all comments.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 4]
```diff
@@ -9,13 +9,16 @@ import NoteTable from "./NoteTable";
 import NewNotePane from "./NewNotePane";
 import DeleteAlert from "./DeleteAlert";
 
+import notesInitialList from "constants/notesInitialList";
+import sortOptions from "constants/sortOptions";
+
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:14]: Please sort all import statements in the correct order. Please refer [Sorting import statements](https://www.bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements) for more details.

**Review by karthiknmenon — COMMENTED** [review]
@josephmathew900 

Please check the suggestions. The `padding` for the section that contains the `Note Listing Table` doesn't match the design.   Please use the same values from Figma.

Also please take care of the following from the next PR - 

1. Please use proper casing for PR titles. Example - `Completed note listing page UI`
2. Please assign PR's for review rather than adding me as a reviewer. 
3. Please attach a description about the PR along with `Fixes #{issueNumber}`.

**Pull request merged**